uk.co.compucorp.civicrm.tasksassignments/js/src/crm-tasks-workflows/controllers/casetype.extended.controller.js

```diff
@@ -124,5 +127,16 @@
 
       return taskOptions.concat(documentOptions);
     }
+
+    /**
+     * Sets the default category for the case type. In this case the default
+     * category is Workflow.
+     */
+    function setCaseTypeDefaultCategory () {
+      var categoryFieldname = 'custom_' + customFieldIds['caseType.category'];
+      var DEFAULT_CATEGORY = 'Workflow';
```

Comment:
How do you know `Workflow` is the default category? Should not we fetch this from backend in any way?